### PR TITLE
🌱 Add option to execute HTML

### PR DESCRIPTION
**Pull request by MikeSpreitzer**
## Summary
This PR introduces a second generation docs-ecutable technology, which extracts the bash from the generated HTML. This has the advantage of having the Jinja processing already done on it.

## Related issue(s)

Fixes #

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -250,11 +250,13 @@ How do we ensure that our documented examples work?  Simple, we 'execute' our do
 
 #### The GitHub Workflow:
 - One example of the GitHub Workflow is located in our {{ config.repo_short_name }} at <a href="{{ config.repo_url }}/blob/{{ config.ks_branch }}/.github/workflows/docs-ecutable-where-resolver.yml">{{ config.repo_url }}/blob/{{ config.ks_branch }}/.github/workflows/docs-ecutable-where-resolver.yml</a>
+- An example workflow using the newer technology is located in our {{ config.repo_short_name }} repo at <a href="{{ config.repo_url }}/blob/{{ config.ks_branch }}/.github/workflows/docs-ecutable-example1.yml">{{ config.repo_url }}/blob/{{ config.ks_branch }}/.github/workflows/docs-ecutable-exammple1.yml</a>
```
Comment (waltforme) [inline docs/README.md:253]: Typo in 'exammple1'.

Comment (MikeSpreitzer) [reply docs/README.md]: Fixed, thanks.

**Comment by MikeSpreitzer** [comment]
The force-push to d8e9f7594 is a rebase plus removing the unnecessary Python installs from the workflow.

**Pull request merged**